henchman/helpers.go

```diff
@@ -21,6 +24,16 @@ func MergeMap(src map[interface{}]interface{}, dst map[interface{}]interface{},
 	}
 }
 
+func MergeLogrusFields(src log.Fields, dst log.Fields, override bool) {
```

Comment:
Can we have a single merge method that takes interfaces ? (More refactoring ;) )